resources/qgis-resource-metadata.xsd

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+###############################################################################
+#
+# Copyright (C) 2017 Tom Kralidis (tomkralidis@gmail.com)
+# Copyright (C) 2017 Angelos Tzotsos (gcpp.kalxas@gmail.com)
+#
+# This source is free software; you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free
+# Software Foundation; either version 2 of the License, or (at your option)
+# any later version.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+###############################################################################
+-->
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:rm="http://qgis.org/resource-metadata/1.0" targetNamespace="http://qgis.org/resource-metadata/1.0" elementFormDefault="qualified" attributeFormDefault="unqualified" version="1.0">
+    <xs:element name="ResourceMetadata" type="rm:ResourceMetadataType">
+        <xs:annotation>
+            <xs:appinfo>QGIS Resource Metadata Schema</xs:appinfo>
+            <xs:documentation>Copyright (C) 2017 Tom Kralidis (tomkralidis@gmail.com)</xs:documentation>
+            <xs:documentation>Copyright (C) 2017 Angelos Tzotsos (gcpp.kalxas@gmail.com)</xs:documentation>
+        </xs:annotation>
+    </xs:element>
+    <xs:complexType name="ResourceMetadataType">
+        <xs:sequence>
+            <xs:element name="identifier" type="xs:string"/>
+            <xs:element name="parentidentifier" type="xs:string" minOccurs="0"/>
+            <xs:element name="language" type="xs:string"/>
+            <xs:element name="type" type="xs:string"/>
+            <xs:element name="title" type="xs:string"/>
+            <xs:element name="abstract" type="xs:string"/>
+            <xs:element name="keywords" type="rm:keywordsType" minOccurs="0" maxOccurs="unbounded"/>
+            <xs:element name="fees" type="xs:string" minOccurs="0"/>
+            <xs:element name="constraints" type="xs:string" minOccurs="0" maxOccurs="unbounded"/>
+            <xs:element name="rights" type="xs:string" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
Can we get an example rights and constraints added to the sample .xml file please?